htdocs/core/class/translate.class.php

```diff
@@ -699,6 +699,10 @@ public function trans($key, $param1 = '', $param2 = '', $param3 = '', $param4 =
 				$str
 			);
 
+			if (preg_match('/%\)/', $str)) {
+				$str = preg_replace('/%\)/', '__percent__)', $str);
+			}
+
```

Comment:
As for the reverse operation you could just do the str_replace directly without the preg_match.
The performance of str_replace is likely a bit better too.

Reply:
thx it's done